server/lib/routes/get-index.js

```diff
@@ -47,6 +49,10 @@ module.exports = function (config) {
       // Note that staticResourceUrl is added to templates as a build step
       staticResourceUrl: STATIC_RESOURCE_URL
     });
+
+    if (req.headers.dnt === '1') {
```

Comment:
Would we be able to extract page load information from Kibana? From what I remember the goal was to see what % of users have DNT enabled?

Reply:
Percentage of flows, not users.

I'm working on the basis that `GET /` is roughly equivalent to that. If you think that's wrong, I'm happy to add an extra log message here for the `else` case, so we compare like with like.